frontend/src/assets/css/variables.scss

```diff
@@ -8,4 +8,5 @@
   --brand-blue-100: #2563eb;
   --text-gray-200: #6b6d7f;
   --hover-blue-100: #1747ae;
+  --black-shadow-color: #393b4d;
```

Comment:
now sure that we have this color in the figma

Reply:
Andrew asked me to take that color into variable